spec/micro/README.md

```diff
@@ -0,0 +1,381 @@
+    counterpartyChannelIdentifier: Identifier, // this is the counterparty's identifier of our chain
+    counterpartyKeyPrefix: CommitmentPrefix,
+    authentication: data, // implementation-specific authentication data
+) {
+    assert(verify(authentication))
+
+    counterparty = Counterparty{
+        channelId: counterpartyChannelIdentifier,
+        keyPrefix: counterpartyKeyPrefix
+    }
+
+    privateStore.set(counterpartyPath(channelIdentifier), counterparty)
+}
+
+// getCounterparty retrieves the stored counterparty identifier
+// given the channelIdentifier on our chain once it is provided
+function getCounterparty(channelIdentifier: Identifier): Counterparty {
+    return privateStore.get(counterpartyPath(channelIdentifier))
+}
+```
+
+The `ProvideCounterparty` method allows for authentication data that implementations may verify before storing the provided counterparty identifier. The strongest authentication possible is to have a valid clientState and consensus state of our chain in the authentication along with a proof it was stored at the claimed counterparty identifier. This is equivalent to the `validateSelfClient` logic performed in the connection handshake.
+A simpler but weaker authentication would simply be to check that the `ProvideCounterparty` message is sent by the same relayer that initialized the client. This would make the client parameters completely initialized by the relayer. Thus, users must verify that the client is pointing to the correct chain and that the counterparty identifier is correct as well before using the lite channel identified by the provided client-client pair.
+
+### IBC Lite
+
+IBC lite will simply provide packet delivery between two chains communicating and identifying each other by on-chain light clients as specified in ICS-02 with application packet data being routed to their specific IBC applications with packet-flow semantics remaining as they were defined in ICS-04. The channelID derived from the clientIDs as mentioned above will tell the IBC router which chain to send the packets to and which chain a received packet came from, while the portID specifies which application on the router the packet should be sent to.
+
+Thus, once two chains have set up clients for each other with specific Identifiers, they can send IBC packets like so.
+
+```typescript
+interface Packet {
+  sequence: uint64
+  timeoutHeight: Height
+  timeoutTimestamp: uint64
+  sourcePort: Identifier // identifier of the application on sender
+  sourceChannel: Identifier // identifier of the client of destination on sender chain
+  destPort: Identifier // identifier of the application on destination
+  destChannel: Identifier // identifier of the client of sender on the destination chain
+}
+```
+
+Since the packets are addressed **directly** with the underlying light clients, there are **no** more handshakes necessary. Instead the packet sender must be capable of providing the correct <client, client> pair.
```

Comment:
I feel like it might be worth noting here, that there might be increased bandwidth for relayers or packet senders, with the added benefit of significantly reducing protocol complexity.